data/campaigns/The_South_Guard/utils/sg_story.cfg

```diff
@@ -191,7 +191,7 @@
         [part]
             background=story/winter.jpg
             # po: Epilogue B
-            {STORY: _"Urza Afalas earned his own mound as well, a small, inconspicuous thing near his old home, where his brothers and he used to live before they were exiled for their crimes. Knowing that the former outlaw leader had helped defend the city against the undead assault, the council of Westin absolved the remaining bandits of their crimes and allowed them to return to their homes. After many years away from home and a difficult and terrifying battle against the undead, most were more than happy to come back in peace and help clean up the town."}
+            {STORY: _"Urza Afalas earned his own mound as well, a small, inconspicuous thing near his old home, where his brothers and he used to live before they were exiled for their crimes. Knowing that the former outlaw leader had helped defend the city against the undead assault, the Council of Westin absolved the remaining bandits of their crimes and allowed them to return to their homes. After many years away from home and a difficult and terrifying battle against the undead, most were more than happy to come back in peace and help clean up the town."}
```

Comment:
And here.

Reply:
Precedents:
https://github.com/wesnoth/wesnoth/blob/6c78d376fe8f4d6b62d6392bef738e500854851d/data/campaigns/The_South_Guard/scenarios/08b_The_Tides_of_War.cfg#L507
https://github.com/wesnoth/wesnoth/blob/6c78d376fe8f4d6b62d6392bef738e500854851d/data/campaigns/The_South_Guard/scenarios/08b_The_Tides_of_War.cfg#L518
https://github.com/wesnoth/wesnoth/blob/6c78d376fe8f4d6b62d6392bef738e500854851d/data/campaigns/The_South_Guard/scenarios/08b_The_Tides_of_War.cfg#L533